### PR TITLE
Refactor/remove console warnings

### PR DESCRIPTION
**Pull request by jiaranda**
# Description
Please include a description of the changes and relevant information. List any dependencies that are required for this change.
## Type of change
Please delete options that are not relevant.
- [x] Refactor: removed console warnings
# How Has This Been Tested?
Please describe the tests that you ran to verify your changes (Like web browser's or mobile devices). Provide instructions so we can reproduce.
- [x] Chrome Desktop - manually

### CODE REVIEW

**Review by dfvasquez — COMMENTED** [review]
Hay unos warnings que faltan por resolver:

image

File: src/containers/navbar.js [hunk 1]
```diff
@@ -2,26 +2,23 @@
...
 // import { CURRENT_USER } from '../graphql/queries/inner_queries';
-import { GET_USER } from '../graphql/queries/users';
 // Bootstrap
 import { Button, Form, FormControl, Navbar, Nav } from 'react-bootstrap';
 
 function EmployeesLinks() {
-    {/* TODO: set correct hrefs for each link */}
     return([
-        <Nav.Link href="/">
+        <Nav.Link href="/" key='available posts'>
```
Comment (cnscheihing) [inline src/containers/navbar.js]:
```suggestion
        <Nav.Link href="/" key='available-posts'>
```

File: src/containers/navbar.js [hunk 2]
```diff
@@ -2,26 +2,23 @@
...
             Ofertas Disponibles
         </Nav.Link>,
-        <Nav.Link href="/">
+        <Nav.Link href="/" key='current applications'>
```
Comment (cnscheihing) [inline src/containers/navbar.js]:
```suggestion
        <Nav.Link href="/" key='current-applications'>
```

**Review by cnscheihing — COMMENTED** [review]
Puse dos suggested-changes jiji. Todo lo demás ok!

**Review by rakarachon — APPROVED** [review]

**Review by dfvasquez — APPROVED** [review]

**Review by cnscheihing — APPROVED** [review]

**Pull request merged**